File: Topic Based Problems/01 Fundamentals/1 Beginner/19 Pounds to Kilograms/C++/PoundstoKilograms.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double pounds;

    cout << "Enter the weight in Pounds: ";
    cin >> pounds;

    cout << fixed << setprecision(2) << "Weight in Kilograms = " << pounds * 0.453592 << endl;

    return 0;
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/19 Pounds to Kilograms/Java/PoundstoKilograms.java
import java.util.Scanner;

public class PoundstoKilograms {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double pounds;

        System.out.print("Enter the weight in Pounds: ");
        pounds = scanner.nextDouble();

        System.out.printf("Weight in Kilograms = %.2f\n", pounds * 0.453592);

        scanner.close();
    }
}
